lib/src/main/kotlin/com/swmansion/starknet/provider/Provider.kt

```diff
@@ -247,6 +248,65 @@ interface Provider {
      */
     fun getStorageAt(contractAddress: Felt, key: Felt): Request<Felt>
 
+    /**
+     * Get a value of storage var.
+     *
+     * Get a value of a storage variable of contract at the provided address.
+     *
+     * @param contractAddress an address of the contract
+     * @param key an address of the storage variable inside contract
+     * @param blockTag The tag of the requested block.
+     *
+     * @throws RequestFailedException
+     */
+    fun getStorageAt(contractAddress: Felt, key: String, blockTag: BlockTag): Request<Felt> {
+        return getStorageAt(contractAddress, selectorFromName(key), blockTag)
```

Comment:
Why do all the newly added methods contain implementations here? Considering this is an interface, shouldn't they be included in `JsonRpcProvided` instead?

Reply:
Initially I thought it could be implemented same as `signV3` in `Account` interface - that's the motivation why I chose to write it this way.
https://github.com/software-mansion/starknet-jvm/blob/c9df29a4e0c420f55211189f741a8c850b63c2cf/lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt#L116-L118
However, let's keep the actual implementation in the class 👍